src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -325,10 +333,19 @@ namespace NServiceBus
     {
         public static NServiceBus.ConventionsBuilder DefiningExpressMessagesAs(this NServiceBus.ConventionsBuilder builder, System.Func<System.Type, bool> definesExpressMessageType) { }
     }
+    [System.ObsoleteAttribute("Message property encryption is released as a dedicated \'NServiceBus.Encryption.Me" +
+        "ssageProperty\' package. Will be treated as an error from version 7.0.0. Will be " +
+        "removed in version 8.0.0.", false)]
     public class EncryptedValue
     {
         public EncryptedValue() { }
+        [System.ObsoleteAttribute("Message property encryption is released as a dedicated \'NServiceBus.Encryption.Me" +
```

Comment:
Nitpick but I have a faint memory that unless the obsolete message on the members is different to provide better guidance we can just remove them since the obsolete message on the class would be the one showed to the users anyway? (cc @SimonCropp )